### PR TITLE
feat: add enableLightningWebSecurityTransforms to rollup-plugin

### PR DESCRIPTION
**Pull request by jdalton**
## Does this pull request introduce a breaking change?

* ✅ No, it does not introduce a breaking change.

## Does this pull request introduce an observable change?

* ✅ No, it does not introduce an observable change.

## GUS work item

W-13014068

### CODE REVIEW

**Review by nolanlawson — APPROVED** [review]

**Comment by nolanlawson** [comment]
@jdalton Approved, please wait to merge, we have some release train stuff going on right now

File: packages/@lwc/rollup-plugin/src/index.ts [hunk 1]
```diff
@@ -309,6 +312,7 @@ export default function lwc(pluginOptions: RollupLwcOptions = {}): Plugin {
                 experimentalDynamicComponent,
                 experimentalDynamicDirective,
                 enableDynamicComponents,
+                enableLightningWebSecurityTransforms,
```
Comment (nolanlawson) [inline packages/@lwc/rollup-plugin/src/index.ts:315]: Please document this in the `README`

**Review by nolanlawson — CHANGES_REQUESTED** [review]
See above

File: packages/@lwc/rollup-plugin/src/index.ts (hunk 1 above)
Comment (jdalton) [reply packages/@lwc/rollup-plugin/src/index.ts:315]: Updated with readme additions :)

Comment (nolanlawson) [reply packages/@lwc/rollup-plugin/src/index.ts:315]: Pinged you on Slack :)

**Review by nolanlawson — CHANGES_REQUESTED** [review]
It just occurred to me we also failed to document this in `@lwc/compiler`. Could you please add a blurb to that README too?

**Comment by jdalton** [comment]
> It just occurred to me we also failed to document this in `@lwc/compiler`. Could you please add a blurb to that README too?

Updated 💪

**Review by nolanlawson — APPROVED** [review]

**Pull request merged**